composables/useNoScroll.ts

```diff
@@ -0,0 +1,6 @@
+export const useNoScroll = () => {
+  const htmlElement = document.querySelector('html');
```

Comment:
Global window.document of document is niet beschikbaar in SSR.

Reply:
Deze methods worden alleen aangeroepen in client, wanneer er een transitie plaats vind.